pkg/server/grpc/options.go

```diff
@@ -0,0 +1,72 @@
+package grpc
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/spf13/pflag"
+
+	addonce "open-cluster-management.io/sdk-go/pkg/cloudevents/clients/addon"
+	clusterce "open-cluster-management.io/sdk-go/pkg/cloudevents/clients/cluster"
+	csrce "open-cluster-management.io/sdk-go/pkg/cloudevents/clients/csr"
+	eventce "open-cluster-management.io/sdk-go/pkg/cloudevents/clients/event"
+	leasece "open-cluster-management.io/sdk-go/pkg/cloudevents/clients/lease"
+	"open-cluster-management.io/sdk-go/pkg/cloudevents/clients/work/payload"
+	grpcauthn "open-cluster-management.io/sdk-go/pkg/cloudevents/server/grpc/authn"
+	grpcoptions "open-cluster-management.io/sdk-go/pkg/cloudevents/server/grpc/options"
+
+	"open-cluster-management.io/ocm/pkg/server/services/addon"
+	"open-cluster-management.io/ocm/pkg/server/services/cluster"
+	"open-cluster-management.io/ocm/pkg/server/services/csr"
+	"open-cluster-management.io/ocm/pkg/server/services/event"
+	"open-cluster-management.io/ocm/pkg/server/services/lease"
+	"open-cluster-management.io/ocm/pkg/server/services/work"
+)
+
+type GRPCServerOptions struct {
+	GRPCServerConfig string
+}
+
+func NewGRPCServerOptions() *GRPCServerOptions {
+	return &GRPCServerOptions{}
+}
+
+func (o *GRPCServerOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&o.GRPCServerConfig, "server-config", o.GRPCServerConfig, "Location of the server configuration file.")
```

Comment:
may use `grpc-server-config`. if we support other servers, we need to expose new parameters. We need to have different loaders to load different server options.